File: src/main/kotlin/com/namor/kurs/service/repository/FilmRepository.java
package com.namor.kurs.service.repository;

import com.namor.kurs.domain.entity.Film;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.rest.core.annotation.RestResource;

@RestResource(rel = "content")
public interface FilmRepository extends JpaRepository<Film, Integer> {

}

File: src/main/kotlin/com/namor/kurs/service/repository/FilmRepository.kt
package com.namor.kurs.service.repository

import com.namor.kurs.domain.entity.Film
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.rest.core.annotation.RestResource

@RestResource(rel = "content")
interface FilmRepository : JpaRepository<Film, Int> {
    fun findAllByNameIgnoreCaseContaining(name: String): List<Film>
}